README.md

```diff
@@ -295,7 +295,7 @@ We use similar hyperparameters as the Stanford Alpaca.
 
 - Install dependency
 ```bash
-pip3 install -e ".[train]"
+pip3 install -r ./fine_tune_requirements.txt
```

Comment:
1. Revert this.
2. Add one tip about `./fine_tune_requirements.txt` under the `Tips` section below

Reply:
ok, thank you for your suggestion.